client/src/pages/admin/merge/MergePeople.tsx

```diff
@@ -497,12 +497,19 @@ const PersonColumn = ({
         <AdvancedSingleSelect
           fieldName="person"
           placeholder="Select a person to merge"
-          value={person}
+          value={
+            person
+              ? { ...person, name: Person.militaryName(person.name) }
+              : person
+          }
           overlayColumns={["name"]}
           overlayRenderRow={PersonSimpleOverlayRow}
           filterDefs={peopleFilters}
           onChange={value => {
-            value?.fixupFields()
+            if (value) {
+              value.name = Person.militaryName(value.name)
+              value.fixupFields()
+            }
```

Comment:
Unnecessary change, please revert.

Reply:
I believe we want to maintain the previous behaviour where the last name is capitalized, right?

Reply:
If we really want to do that, something like:
```javascript
           objectType={Person}
           valueKey="name"
+          valueFunc={(v, k) => k === "name" ? Person.militaryName(v?.[k]) : v?.[k]}
           fields={Person.allFieldsQuery}
           addon={PEOPLE_ICON}
```
would be a better solution, but we would have to apply this to all AdvancedSingleSelect's for person.